assets/js/html5-details-polyfill.js

```diff
@@ -1,8 +1,9 @@
 var ua = window.navigator.userAgent;
 var isIE = /MSIE|Trident/.test(ua);
+var isEdge = (window.navigator.appName === "Netscape");
```

Comment:
This also matches FF and chrome and seems to break FF
https://www.w3schools.com/jsref/prop_nav_appname.asp